src/lib/CombatCalc.ts

```diff
@@ -1590,7 +1609,7 @@ export default class CombatCalc {
     const dist = this.getDistribution();
     const hist = dist.asHistogram();
     const max = dist.getMax();
-    if (max === 0) {
+    if (dist.getExpectedDamage() === 0) {
```

Comment:
Previous condition doesn't match if you're splashing on an enemy so the ttk calculation never finishes